lib/markdown2.py

```diff
@@ -2037,15 +2037,37 @@ def _add_footnotes(self, text):
                 '<hr' + self.empty_element_suffix,
                 '<ol>',
             ]
+
+            try:
+                footnote_title = self.footnote_title
+            except AttributeError:
+                footnote_title = 'title="Jump back to footnote %d in the text."'
```

Comment:
Should we make this even more specific, thinking title being the actual title text vs having the end user make sure to include `title=""`? Isn't the `title` arg always going to be required?

My thought would be `footnote_title = 'Jump back to footnote %d in the text.'`